spring/narayana-spring-boot/pom.xml

```diff
@@ -73,14 +77,19 @@
     </dependencyManagement>
 
     <dependencies>
+        <dependency>
+            <groupId>org.javassist</groupId>
+            <artifactId>javassist</artifactId>
+            <version>3.25.0-GA</version>
+        </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-jpa</artifactId>
         </dependency>
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-artemis</artifactId>
-        </dependency>
+        </dependency>      
```

Comment:
FYI a bit of extra whitespace at the end of this line that should be able to be removed I expect

Reply:
Thanks Tom, I will do it.